generator/src/test/java/com/scottlogic/deg/generator/cucumber/steps/GeneralTestStep.java

```diff
@@ -166,11 +166,25 @@ public void theFollowingDataShouldNotBeContainedInActual(List<Map<String, String
 
         Assert.assertThat(
             "Exceptions thrown during generation",
-            testHelper.getThrownExceptions(),
+            cucumberTestHelper.getThrownExceptions(),
             empty());
         Assert.assertThat(data.generatedData, new RowsAbsentMatcher(data.expectedData));
     }
 
+    @Given("^the data requested is (violating|validating)$")
```

Comment:
Follow the same pattern as generationType/generationStrategy, see `TypeRegistryConfiguration`

Reply:
Am doing so but for the record I think this is cumbersome and over the top for what should be a very simple cucumber step.